--- gnome-nettool-0.99.2/configure.in.orig	2004-08-18 22:42:16.000000000 +0200
+++ gnome-nettool-0.99.2/configure.in	2004-08-20 13:28:55.038625736 +0200
@@ -60,7 +60,7 @@
 GETTEXT_PACKAGE=gnome-nettool
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE",[The gettext package])
-ALL_LINGUAS="az bn ca cs da de el en_CA en_GB es fr ga gl gu he hu it ja ko ms nl no pa pl pt pt_BR ru sq sr sr@Latn sv tr uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="az bn ca cs da de el en_CA en_GB es fr ga gl gu he hu it ja ko ms nb nl pa pl pt pt_BR ru sq sr sr@Latn sv tr uk vi wa zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 AC_SUBST(CFLAGS)
