diff -aurN gnome-nettool-2.14.1.orig/src/gnome-nettool.desktop.in gnome-nettool-2.14.1/src/gnome-nettool.desktop.in
--- gnome-nettool-2.14.1.orig/src/gnome-nettool.desktop.in	2006-04-03 21:11:53.000000000 +0200
+++ gnome-nettool-2.14.1/src/gnome-nettool.desktop.in	2006-04-10 19:30:14.225375048 +0200
@@ -6,7 +6,7 @@
 Icon=gnome-nettool.png
 Terminal=false
 Type=Application
-Categories=Application;System;Utility;GNOME;GTK;
+Categories=GTK;GNOME;System;Utility;
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-nettool
