diff -urN aa/gnome-nettool-0.99.1/src/gnome-nettool.desktop.in gnome-nettool-0.99.1/src/gnome-nettool.desktop.in
--- aa/gnome-nettool-0.99.1/src/gnome-nettool.desktop.in	2004-06-04 13:24:39.000000000 +0200
+++ gnome-nettool-0.99.1/src/gnome-nettool.desktop.in	2004-07-24 01:12:31.324922288 +0200
@@ -6,7 +6,7 @@
 Icon=gnome-nettool.png
 Terminal=false
 Type=Application
-Categories=Application;System;Utility;
+Categories=GTK;GNOME;System;Utility;
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-nettool
